src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/vcf/VcfFuncotationFactory.java

```diff
@@ -87,6 +87,12 @@
      */
     private static final String ID_FIELD_NAME = "ID";
 
+    /**
+     * The name of the additional FILTER status field to add to VCF annotations to preserve the FILTER status of the original (data source)
+     * variant.
+     */
+    private static final String FILTER_FIELD_NAME = "FILTER";
```

Comment:
Isn't there a VCFConstants constant for this?

Reply:
Not that I can see.  There's one that's close, but contains comments which won't work:

```public static final String FILTER_HEADER_START = "##FILTER";```